### ISSUE TITLE
Fix camera computation

### ISSUE THREAD

**Issue by hozuki** [OpenMLTD/MLTDTools#18]
Coordinate computation in camera motion is still buggy. May be because of LHS to RHS transformation.

**Comment by hozuki** [comment]
There are some strage values in camera curves.

For example, look at frame ~5602 of `alstar`. Comparing to the reference visual, the converted camera is at a different position, with completely different Y (i.e. below the stage, XOZ plane).

Coverted:

```plain
pos = (-11.06979, 3.231802, -34.16661)
rot = (-9.7438932, 164.6800598, 0.7779374)
```

To reproduce the reference visual:

```plain
pos = (-11.06979, ~8.0, -34.16661)
rot = (~-2.74, 164.6800598, 0.7779374)
```

**Comment by hozuki** [comment]
At least starting from Glow Map, scenario introduced procedural camera control (type 58).

**Comment by hozuki** [comment]
General camera orientation was fixed in commits a17b8b7ceabbafd6338b1dcef4f9a48fd18328a7 and 8a0a2de03b4be808a1cb886186f1efa4e4050f43.
It requires more samples to investigate procedural camera control.
